### PR TITLE
Klimakhovich - first hometask

### PR DESCRIPTION
**Pull request by Vladonblch**

### CODE REVIEW

**Comment by Vladonblch** [comment]
@AlexOreshkevich please review

**Review by AlexOreshkevich — CHANGES_REQUESTED** [review]
Please provide unit test and other changes according to our voice conversation

File: app/src/main/java/by/academy/jc/klimakhovich/MyClass.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+public class MyClass {
+
+    public String greeting() {
+        return "Hello Vlad you are amazing!";
+    }
+
+    public static void main(String[] args) {
+        System.out.println(new MyClass().greeting());
+    }
+
```
Comment (AlexOreshkevich) [inline app/src/main/java/by/academy/jc/klimakhovich/MyClass.java:13]: Nit-picky: unnecessary blank lines

File: app/src/test/java/by/academy/jc/klimakhovich/TestMyClass.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class TestMyClass {
+
+    @Test
+    void firstTestClass() {
+        MyClass classUnderTest = new MyClass();
+        assertNotNull(classUnderTest.greeting(), "MyClass should have a greeting");
```
Comment (AlexOreshkevich) [inline app/src/test/java/by/academy/jc/klimakhovich/TestMyClass.java:13]: I was asking for strong checks, like `assertEquals(expected, actual)`.

File: app/src/test/java/by/academy/jc/klimakhovich/TestMyClass.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class TestMyClass {
+
+    @Test
+    void firstTestClass() {
+        MyClass classUnderTest = new MyClass();
```
Comment (AlexOreshkevich) [inline app/src/test/java/by/academy/jc/klimakhovich/TestMyClass.java:12]: Store `classUnderTest` variable as a unit class property and initialize it in some of the two `@Before` clauses available.

**Review by AlexOreshkevich — APPROVED** [review]
lgtm

**Pull request closed without merging**